src/components/Login/Login.jsx

```diff
@@ -0,0 +1,97 @@
+import { useState, useContext } from 'react';
+import { Link } from 'react-router-dom';
+import { Context } from 'src/index';
+import Snackbars from 'src/Snackbars/Snackbars';
+import { validationObject } from 'src/helper/helper-validate';
+import logo from 'src/logos/mainLogo.svg';
+import icon from 'src/logos/buildings.svg';
+import './style.scss';
+
+const Login = () => {
```

Comment:
and why did you flood the authorization and registration again? They've already been checked.

Reply:
fill in only new components or those where there have been changes